src/app/viewer/viewer-form-blocks/ShortAnswer.tsx

```diff
@@ -9,14 +9,25 @@ export default function ShortAnswer({block_structure, updateFormDataBlock}: {blo
 
     const [inputValue, setInputValue] = useState('')
 
+    function debounce(func, timeout = 300){
```

Comment:
can we move it to the lib folder, so that it can be reused throughout the project